ext/gentoo/puppet/puppet.conf

```diff
@@ -0,0 +1 @@
+../../../conf/puppet.conf
```

Comment:
Same comment on symlinks as before. Git for Windows doesn't know how to deal with Symlinks.